torch/csrc/jit/codegen/cuda/kernel.cpp

```diff
@@ -586,10 +562,40 @@ void runKernel(
     kernel_args.push(output);
   }
 
+  Fusion* fusion = entry->fusion_.get();
+  FusionGuard fg(fusion);
+  EvaluationContext eval_context(fusion);
+  for (int i = 0; i < (int)inputs.size(); i++) {
+    if (inputs[i].isTensor()) {
+      ExtractSizeStride ess(inputs[i].toTensor(), broadcasted_shape);
+      int nDims = ess.sizes.size();
+      TensorView* tv = fusion->inputs()[i]->as<TensorView>();
+      for (int j = 0; j < nDims; j++) {
+        eval_context.bind(tv->getRootDomain()[j]->extent(), ess.sizes[j]);
+      }
+    }
+  }
+  auto expr_eval_fn = [&](LaunchConfigType type) -> int {
```

Comment:
nit: 
1. please empty line before the lambda definition
2. `-> int` is not required